core/examples/fulltextsearch.js

```diff
@@ -20,7 +20,6 @@ Ext.onReady(function() {
             coordsRecenterZoom: 11,
             widgetOptions: {
               tooltipEl: 'search-tip',
-              projectionCodes: [21781, 4326],
```

Comment:
If we are really sure that we want to move ``projectionCodes`` config from the widget to the plugin, you need to add an entry in CGXP's changelog.